.github/workflows/stale.yml

```
@@ -0,0 +1,28 @@
+# https://github.com/actions/stale
+name: 'Close stale issues and PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+permissions:
+  issues: write
+  pull-requests: write
+
+concurrency:
+  group: lock
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-message: 'This issue is stale because it has been open 60 days with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          days-before-issue-stale: 60
+          days-before-issue-close: 15
+          close-issue-message: 'This issue was closed because it has been stalled for 15 days with no activity. If this issue is still valid please re-open'
+
+          stale-pr-message: 'This PR is stale because it has been open 90 days with no activity. Remove stale label or comment or this will be closed in 15 days.'
+          days-before-pr-stale: 90
+          days-before-pr-close: 15
+          close-pr-message: 'This PR was closed because it has been stalled for 15 days with no activity. If this PR is still valid please re-open'
```

Just a few style suggestions here but feel free to reformulate. Since they'll be proper comments I guess we have space for more text :)

```suggestion
          stale-issue-message: 'This issue was marked stale because it has been open 60 days with no activity. Please remove the stale label or comment on this issue. Otherwise, this will be closed in 15 days.'
          days-before-issue-stale: 60
          days-before-issue-close: 15
          close-issue-message: 'This issue was closed because it has been marked stale for 15 days with no activity. If this issue is still valid, please re-open.'

          stale-pr-message: 'This PR was marked stale because it has been open 90 days with no activity. Please remove the stale label or comment on this PR. Otherwise, this will be closed in 15 days.'
          days-before-pr-stale: 90
          days-before-pr-close: 15
          close-pr-message: 'This PR was closed because it has been marked stale for 15 days with no activity. If this PR is still valid, please re-open.'
```

---

Done. Thanks.